src/test/java/com/wikia/webdriver/testcases/desktop/mcfootertests/NonEnAnonMixedContentFooterTests.java

```diff
@@ -133,6 +263,21 @@ public void userIsTakenToDiscussionsPostViewAfterClickOnPost() {
     Assertion.assertTrue(url.contains(".wikia.com/d/"));
   }
 
+  @RunOnly(language = "szl")
+  @Execute(onWikia = "mediawiki119")
+  @Test
+  public void userIsTakenToDiscussionsPostViewAfterClickOnPostSzl() {
+    new MixedContentFooter().openWikiMainPage()
+        .scrollToMCFooter()
+        .getDiscussionsCard()
+        .scrollToDiscussions()
+        .clickDiscussionsPost();
+
+    String url = driver.getCurrentUrl();
+    Assertion.assertTrue(url.contains(".wikia.com/szl/d/"));
```

Comment:
`url` is used only once, so I guess we could use it inline i.e. `driver.getCurrentUrl().contains()`